jenkins/azure-pipelines.yml

```diff
@@ -8,13 +8,41 @@ trigger:
 pr:
 - '*'
 
+resources:
+  repositories:
+  - repository: maccore
+    type: github
+    name: xamarin/maccore
+    ref: refs/heads/main
+    endpoint: xamarin
+
+variables:
+  -group: xamops-azdev-secrets
```

Comment:
Missing space between - and group. should be '- group:'

Reply:
Good catch!